lib/perlfaq4.pod

```diff
@@ -2197,10 +2197,9 @@ such as C<each()>.
 (contributed by brian d foy)
 
 To sort a hash, start with the keys. In this example, we give the list of
-keys to the sort function which then compares them ASCIIbetically (which
-might be affected by your locale settings). The output list has the keys
-in ASCIIbetical order. Once we have the keys, we can go through them to
-create a report which lists the keys in ASCIIbetical order.
+keys to the sort function which then compares them strings. The output list 
```

Comment:
"compares them strings" -> "compares them as strings"

Reply:
Note that without "use locale", the comparison order may be better described as ASCIIbetical, and not how one would expect locale-based string comparisons. Not that this answer necessarily needs to detail that.